dali/python/nvidia/dali/ops.py

```diff
@@ -21,7 +21,7 @@
 from nvidia.dali import backend as b
 from nvidia.dali.types import _type_name_convert_to_string, _type_convert_value, \
         _vector_element_type, _bool_types, _int_types, _int_like_types, _float_types, \
-        DALIDataType, CUDAStream, Constant
+        DALIDataType, CUDAStream, ScalarConstant as _Constant
```

Comment:
...actually, we should add more underscores and import aliases here, because it pollutes `ops` module.